pkg/permissions/claims.go

```diff
@@ -26,20 +25,24 @@ const (
 
 	// RefreshTokenAudience is the audience field of JWT for refresh tokens
 	RefreshTokenAudience = "refresh"
-
-	// TokenValidityDuration is the duration where a token is valid in seconds (1 week)
-	TokenValidityDuration = int64(7 * 24 * time.Hour / time.Second)
 )
 
+// TokenValidityDuration is the duration where a token is valid in seconds (1 week)
+var TokenValidityDuration = 7 * 24 * time.Hour
+
 // Claims is used for JWT used in OAuth2 flow and applications token
 type Claims struct {
 	jwt.StandardClaims
 	Scope string `json:"scope,omitempty"`
 }
 
+// IssuedAtTime returns a time.Time struct of the IssuedAt field
+func (claims *Claims) IssuedAtTime() time.Time {
+	return time.Unix(claims.IssuedAt, 0)
+}
+
 // Expired returns true if a Claim is expired
 func (claims *Claims) Expired() bool {
-	now := crypto.Timestamp()
-	validUntil := claims.IssuedAt + TokenValidityDuration
-	return validUntil < now
+	validUntil := claims.IssuedAtTime().Add(TokenValidityDuration)
+	return validUntil.Before(time.Now())
```

Comment:
I think we are OK since we compare `time.Time` of the same locations. I'll force the `UTC()` though to be clearer on the `IssuedAtTime()` method (renamed to `IssuedAtUTC()`)